.vsts-dotnet.yml

```diff
@@ -34,7 +34,7 @@ variables:
     - name: SourceBranch
       value: ''
   - name: EnableReleaseOneLocBuild
-    value: true
+    value: false
```

Comment:
We'll need to be super careful on the merge-forward to leave this on in the later branches. I wonder if we can add a comment to this line to cause a merge conflict?